package/MDAnalysis/core/groups.py

```diff
@@ -136,6 +136,28 @@ def __add__(self, other):
 
         return self.__class__(np.concatenate([self._ix, o_ix]), self._u)
 
+    def __radd__(self, other):
```

Comment:
Yeah I like it.  I think previously I've used `reduce(lambda x, y: x+y, ags)` or something stupid, so using sum is much better